style.css

```
@@ -1,8 +1,6 @@
 .markdown-alert {
-    padding: 1em;
+    padding: 0 1em !important;
```

This is what GitHub itself uses, though maybe it wouldn't even match correctly due to the weird CSS? Looks good to me this way though.

```suggestion
    padding: 0.5rem 1rem !important;
```